tacacs_plus/__init__.py

```diff
@@ -492,14 +661,15 @@ def send(self, body, seq_no=1):
             )
         raise socket.timeout
 
-    def authenticate(self, username, password,
+    def authenticate(self, username, password, priv_lvl=TAC_PLUS_PRIV_LVL_MIN,
```

Comment:
Same comment goes for this parameter; we're passing this parameter in, but doing nothing with in terms of packet building.